src/ProviderPR/SFA.DAS.ProviderPR.UnitTests/Application/Requests/Commands/AddAccount/WhenEmailInRequestIsNotOwner.cs

```diff
@@ -88,12 +88,4 @@ public async Task Handle_EmailInRequestIsNotOwner_InformationNotificationsSentTo
 
         _providerRelationshipsApiRestClient.Verify(c => c.PostNotifications(It.Is<PostNotificationsCommand>(c => c.Notifications.Count(n => n.TemplateName == NotificationConstants.AddAccountInformationTemplateName) == 1), It.IsAny<CancellationToken>()));
     }
-
-    [Test]
-    public async Task Handle_EmailInRequestIsNotOwner_NotificationNotSentToOtherOwnerWithNotificationPreferenceSetToFalseAsync()
-    {
-        await sut.Handle(createRequestCommand, CancellationToken.None);
-
-        _providerRelationshipsApiRestClient.Verify(c => c.PostNotifications(It.Is<PostNotificationsCommand>(c => !c.Notifications.Any(n => n.EmailAddress == _otherOwnerMemberWithNotificationTurnedOff.Email)), It.IsAny<CancellationToken>()));
-    }
```

Comment:
Why was this test removed?

Reply:
It was redundant, a similar test exists just above